Remotion/Web/Development.WebTesting.IntegrationTests/FileDownloadTest.cs

```diff
@@ -192,6 +192,18 @@ public void TestDownloadOpensInNewWindow_DownloadTimeoutExceeded_WithUnknownFile
           restartBrowserTask.GetAwaiter().GetResult();
         }
       }
+      else

```

Comment:
For continuity's sake: I asked why the extra branch here for Edge and Firefox and you told me that they don't need the restart browser thingy, so there's that